homeassistant/components/lock/zwave.py

```diff
@@ -28,9 +28,37 @@
 DANALOCK_V2_BTZE = 0x2
 POLYCONTROL_DANALOCK_V2_BTZE_LOCK = (POLYCONTROL, DANALOCK_V2_BTZE)
 WORKAROUND_V2BTZE = 'v2btze'
+WORKAROUND_DEVICE_STATE = 'state'
+# Kwikset 914TRL ZW500
+KWIKSET = 0x0090
+ZW500_914TRL = 0x440
```

Comment:
Why all these constants that are only used once ? We should just inline them at the one place where they are used.

Reply:
I was just mirroring the pattern that was already in there for the Polycom workaround (and in other zwave components that have workaround sections) But, if it's that much of a problem I can change it to inline the ids in the device mapping dict.